3741/1/top_gems.rb

```diff
@@ -0,0 +1,31 @@
+require 'yaml'
+require 'optparse'
+require_relative 'lib/read_yaml'
+require_relative 'lib/gem_data_reader'
+require_relative 'lib/show_table'
+
+# main run class
+class TopGems
+  # :reek:NestedIterators
```

Comment:
reek should pass here:

you can rewrite the code using explicit notation
```
opts = OptionParser.new
opts.on('--top=', '--top') { |value| @option_top = value.to_i }
```
or better:
```
option_parser = OptionParser.new
option_parser.on('--top=', '--top') { |value| @option_top = value.to_i }
option_parser.parse!
```

Reply:
'better' sentence i used